documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -9,57 +13,92 @@
-* Use thes the following properties to configure Apache Connect instance
+* An Avro converter that you can configure in Kafka Connect workers. This converter maps Kafka Connect schemas to Avro schemas. The converter then uses the Avro schemas to serialize the message keys and values into Avro's compact binary form.
 
-[source]
+* An API/Schema registry that tracks:
++
+** Avro schemas that are used in Kafka topics
+** Where the Avro converter sends the generated Avro schemas
+
++
+Since the Avro schemas are stored in this registry, each message need only include a tiny _schema identifier_.
+This makes each message even smaller. For an I/O bound system like Kafka, this means more total throughput of the producers and consumers.
+
+* Avro _Serdes_ (serializers and deserializers) for Kafka producers and consumers.
+Any Kafka consumer applications that you write to consume change event records can use Avro Serdes to deserialize the change event records.
+
+To use the {registry} with {prodname}, you must add {registry} converters and their dependencies to the Kafka Connect container image that you are using for running {prodname}.
+
+[NOTE]
+====
+The {registry} project also provides a JSON converter. This converter combines the advantage of less verbose messages with human-readable JSON. Messages do not contain the schema information themselves, but only a schema ID.
+====
+
+ifdef::community[]
+Another option is using the Confluent schema registry, which is described later.
+endif::community[]
+
+// Type: concept
+// Title: Overview of deploying a {prodname} connector that uses Avro serialization
+[id="overview-of-deploying-a-debezium-connector-that-uses-avro-serializatioin"]
+== Overview
+
+To deploy a {prodname} connector that uses Avro serialization, there are three main tasks: 
+
+ifdef::community[]
+. Deploy an link:https://github.com/Apicurio/apicurio-registry[{registry-name-full}] instance.
+endif::community[]
+ifdef::product[]
+. Deploy a link:{LinkServiceRegistryGetStart}[{registry-name-full}] instance by following the instructions in {NameServiceRegistryGetStart}.
+endif::product[]
+
+ifdef::community[]
+. Install the Avro converter from link:https://repo1.maven.org/maven2/io/apicurio/apicurio-registry-distro-connect-converter/{apicurio-version}/apicurio-registry-distro-connect-converter-{apicurio-version}-converter.tar.gz[the installation package] into Kafka Connect's _libs_ directory or directly into a plug-in directory.
+endif::community[]
+ifdef::product[]
+. Install the Avro converter by downloading the {prodname} link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=red.hat.integration&downloadType=distributions[Service Registry Kafka Connect] zip file and extracting it into the {prodname} connector's directory.
+endif::product[]
+
+. Configure a Kafka Connect instance with the following property settings: 
```

Comment:
Can you add a sentence after the listing like this: Alternatively, you can also configure specific connectors to use the Avro converter.

Reply:
Can the word "also" be removed from the sentence that you want to add?
And should this additional sentence be indented to align with the listing? I think so, but I'm not certain.

Reply:
Yes, you can remove the "also". Re indentiation, I don't know, perhaps even just in brackets after "property settings (alternatively...)".